VideoDuplicateFinder.Windows/ViewModels/DuplicateViewModel.xaml

```diff
@@ -21,11 +21,11 @@
     <Grid Margin="10">
         <Grid.ColumnDefinitions>
             <ColumnDefinition Width="Auto" />
-            <ColumnDefinition Width="Auto" />
```

Comment:
The reason for size auto was that when we enable more than 1 thumbnail taken they get merged so the thumbnail widht is unknown. A fixed width would screw this up

Reply:
The accuracy is good but two thumbnails can eliminate most of the false reports. So I'm planning to make this become an option for the user in the (near) future. So I cannot merge your pull request because of it